hazelcast/src/main/java/com/hazelcast/config/CacheSimpleConfig.java

```diff
@@ -1042,23 +1042,47 @@ public void readData(ObjectDataInput in) throws IOException {
                 /**
                  * Expiry policy type for the {@link javax.cache.expiry.CreatedExpiryPolicy}.
                  */
-                CREATED,
+                CREATED(0),
```

Comment:
Did we put documentation of the possible values in the protocol doc? This may be valuable info for new client implementors.